content/api/commands/selectfile.md

```diff
@@ -0,0 +1,276 @@
+---
+title: selectFile
+---
+
+Attaches a file or files to an HTML5 input element or simulates dragging a file
+or files into the browser.
```

Comment:
```suggestion
Selects a file or files to be uploaded to an HTML5 input element or simulates dragging a file
or files into the browser.
```

Reply:
Going for a slightly different edit here:

> Selects a file or files to attach to an HTML5 input element or simulates dragging a file or files into the browser.

I don't necessarily think we need to avoid the word attach, since it's still more descriptive of what's actually happening than 'upload' is.

Reply:
It's replacing 'attach' with 'to be uploaded' not directly the verb 'upload' since it's not uploaded yet. I think upload is more descriptive. If you look at the docs we sighted, like mdn - it uses attach 2 times and upload 35 times for instance. When I search for this feature in google I always type 'upload files'. I don't think attach is more descriptive than upload. @cowboy thoughts?